src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
+	private ArrayList<Integer> numberOfChars;
+	private ArrayList<String> IIN;
+	
+	public String getName(){
+		return name;
+	}
+	
+	public ArrayList<Integer> getNumberOfChars(){
```

Comment:
Declarations should use Java collection interfaces such as "List" rather than specific implementation classes such as "ArrayList". We can change it here and in all related places.